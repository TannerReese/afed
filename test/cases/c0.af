# This is the first test
# It is a valid file and should contain no errors

d + 0.5 ^ (-z / 2) = previous stuff # Result without label

wood / d =

z : -9.87 + 5.3412 =

	x : y // 10 + d // 2 % 0.5 ^ (-
	z / 2) =# A result should be printed here

# Label with no result
 y : z ^ 2 + z^3 * (z -
	1 /
	(-z)^0.5
) # Comment here should be fine

 d : -3.1245#Comment with no leading whitespace

(wood * 2 + z * y + 2 + 2) =
wood * z =
wood : z + y - 3 = # Variable

