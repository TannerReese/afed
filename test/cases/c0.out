# This is the first test
# It is a valid file and should contain no errors

d + (1/2) ^-z/2 = -3.102840 # Result without label

wood / d = -152.757204 

z : -9.87 + 5.3412 = -4.528800 

	x : y // 10 + d // 2 % 0.5 ^ (-
	z / 2) = 47.873226 # A result should be printed here

# Label with no result
 y : z ^ 2 + z^3 * (z -
	1 /
	(-z)^0.5
) # Comment here should be fine

 d : -3.1245#Comment with no leading whitespace

(wood * 2 + z * y + 2 + 2) = -1237.067089 
wood * z = -2161.550429 
wood : z + y - 3 = 477.289884 # Variable

